packages/gatsby-theme-bodiless/src/dist/Page.tsx

```diff
@@ -71,6 +71,7 @@ const ShowDesignKeys = (
 const Page: FC<PageProps> = observer(({ children, ui, ...rest }) => {
   const { PageEditor: Editor, ContextWrapper: Wrapper } = getUI(ui);
   if (process.env.NODE_ENV === 'development') {
+    console.log('edit page');
```

Comment:
Another log.

Reply:
@lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!